trelloscrum.js

```diff
@@ -257,7 +257,7 @@
     }
 
     //track card-titles
-    new TextChangeListener("a", "a.js-card-name", function (elm) {
+    new TextChangeListener("span", "span.js-card-name", function (elm) {
```

Comment:
Is het mogelijk om gewoon alleen naar de class-name te kijken, dus dat het niet uitmaakt of het een `<a>` of `<span>` of whatever is?